src/main/java/org/folio/rest/impl/PatronBlockConditionsAPI.java

```diff
@@ -40,16 +51,24 @@ public void putPatronBlockConditionsByPatronBlockConditionId(
     Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    log.debug("putPatronBlockConditionsByPatronBlockConditionId:: parameters " +
+        "patronBlockConditionId: {}, lang: {}, entity: {}, okapiHeaders: {}",
+      () -> patronBlockConditionId, () -> lang, () -> logAsJson(entity),
+      () -> logOkapiHeaders(okapiHeaders));
+
     Errors errors = validateEntity(entity);
     if (errors != null) {
+      log.info("putPatronBlockConditionsByPatronBlockConditionId:: entity is invalid. Errors: {}",
```

Comment:
```suggestion
      log.warn("putPatronBlockConditionsByPatronBlockConditionId:: entity is invalid. Errors: {}",
```

Reply:
I usually only log exceptions with warn in this PR